lib/register_screen.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:transportation_app/widgets/customized_text.dart';
+import 'widgets/customized_elevatedbutton.dart';
+import 'widgets/customized_textfieldbutton.dart';
+
+class RegisterScreen extends StatefulWidget {
+  const RegisterScreen({Key? key}) : super(key: key);
+
+  @override
+  State<RegisterScreen> createState() => _RegisterScreenState();
+}
+
+class _RegisterScreenState extends State<RegisterScreen> {
+
+  //3 adet Textfield tanımladım.
+
+  final usernameController = TextEditingController();
+  final passwordController = TextEditingController();
+  final emailController = TextEditingController();
+
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SingleChildScrollView(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            const SizedBox(
+              height: 100,
+            ),
+
+            //Logoyu burda tanımladım.
+
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: SvgPicture.asset("images/transportation_miniicon.svg"),
+            ),
+
+            //Register yazısını burada tanımladım.
+
+             Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: customizedText(text: "Register", textcolor: const Color(0xff191849),)
+            ),
+
+            //Username butonunu burada tanımladım.
+
+            CustomizedTextField(
+              myController: usernameController,
+              hintText: "Username",
+              helperText: "Enter your username",
+              isPassword: false,
+              hintColor: const Color(0xffCCCBFB),
```

Comment:
isPassword için detaylı bir yorum bıraktım. Onu inceleyebilirsin

Reply:
isPassword detayı dediğiniz gibi düzenlendi hocam.